src/components/hero.tsx

```diff
@@ -0,0 +1,26 @@
+import { Component, Prop } from 'vue-property-decorator';
+import { VueComponent } from '~/vue-component';
+
+interface HeroProps {
+  title: string;
+  subtitle?: string;
+}
+
+@Component
+export default class HeroComponent extends VueComponent<HeroProps> {
+  @Prop({ required: true })
+  title!: string;
+
+  @Prop()
+  subtitle?: string;
+
+  render() {
+    return <v-card>
```

Comment:
I'm more surprised than anything that babel compiles without throwing errors about parens around multi-line jsx.  I guess I need more training [=

Reply:
You can ask @davidthor for a deep dive on this if you're interested. He had way too much fun with this

Reply:
I had just the right amount of fun, thank you. 😛